yt/frontends/amrex/data_structures.py

```diff
@@ -1497,8 +1495,8 @@ def __init__(self, ds, dataset_type="boxlib_native"):
         for key, val in self.warpx_header.data.items():
             if key.startswith("species_"):
                 i = int(key.split("_")[-1])
-                charge_name = "particle%.1d_charge" % i
```

Comment:
this is a weird one. As far as I understand `%.1d` is accepted but meaningless, and simply equivalent to `%d`.